packages/teams-js/src/internal/utils.ts

```diff
@@ -37,6 +37,22 @@ function validateHostAgainstPattern(pattern: string, host: string): boolean {
   return false;
 }
 
+export function getDomainsFromCDN(): void {
+  fetch(validDomainsCdnEndpoint)
```

Comment:
does fetch cause any problems in SSR scenarios where this is no window object? Do we need to surround this with some sort of SSR check?

Reply:
Good catch! I had this wrapped in a `if (fetch)` call previously which would've defaulted back to the fallback in SSR scenarios, I will test this new code to ensure SSR isn't broken.